### PR TITLE
Rename command

### PR DESCRIPTION
**Pull request by jamesd**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -172,3 +172,18 @@ func (conf *GrueConfig) DeleteAccount(name string) error {
...
+
+func (conf *GrueConfig) RenameAccount(old, new string) error {
+	if conf.Accounts == nil {
+		return fmt.Errorf("%s: account does not exist", old)
+	}
+	if _, ok := conf.Accounts[old]; !ok {
+		return fmt.Errorf("%s: account does not exist", old)
+	}
+	if _, ok := conf.Accounts[new]; ok {
+		return fmt.Errorf("%s: account already exist", new)
```
Comment (c-14) [inline config/config.go]: minor nit, exists

**Review by c-14 — APPROVED** [review]
Otherwise lgtm

File: config/config.go (hunk 1 above)
Comment (jamesd) [reply config/config.go]: Will fix.

**Comment by jamesd** [comment]
Fixed typo.

**Comment by c-14** [comment]
Merged, thanks

**Pull request merged**